clients/vault/src/oracle/agent.rs

```diff
@@ -215,11 +221,15 @@ mod tests {
 	#[ntest::timeout(1_800_000)] // timeout at 30 minutes
 	#[serial]
 	async fn test_get_proof_for_current_slot() {
+		// let it run for a few seconds, making sure that the other tests have successfully shutdown
+		// their connection to Stellar Node
+		sleep(Duration::from_secs(2)).await;
+
 		let shutdown_sender = ShutdownSender::new();
 
 		// We use a random secret key to avoid conflicts with other tests.
 		let agent = start_oracle_agent(
-			get_test_stellar_relay_config(true),
+			specific_stellar_relay_config(true, 0),
```

Comment:
Just curious, why do we want to get always the first node choice here? and then on the other test always the [second](https://github.com/pendulum-chain/spacewalk/pull/475/files#diff-732d3f237cd7cd9cf03fc64a9ae387ce0e80ad0928e995a3b9e61df4a4c24429R266).

Reply:
ah, sometimes an error `already connected peer` happens. I tried connecting to different nodes just to see if it works :crossed_fingers:. The next best thing is to actually connect with different stellar accounts. Currently we only have 1 test account to play with.